basic-data-structures/check-if-an-object-has-a-property.js

```diff
@@ -0,0 +1,35 @@
+let users = {
+  Alan: {
+    age: 27,
+    online: true
+  },
+  Jeff: {
+    age: 32,
+    online: true
+  },
+  Sarah: {
+    age: 48,
+    online: true
+  },
+  Ryan: {
+    age: 19,
+    online: true
+  }
+};
+
+function isEveryoneHere(userObj) {
+  // Only change code below this line
+  let namesOfUsers = ["Alan", "Jeff", "Sarah", "Ryan"];
+
+  for (let i = 0; i < namesOfUsers.length; i++) {
+    let names = namesOfUsers[i];
```

Comment:
use name in singular because each iteration you are talking abount only one user.